.travis.yml

```diff
@@ -1,15 +1,5 @@
 language: node_js
-node_js:
-- 4
```

Comment:
[Travis now looks for `.nvmrc` and uses that defined version](https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Using-.nvmrc) so you don't have to keep it updated in two places :raised_hands: